apps/storybook/src/Utilities.stories.mdx

```diff
@@ -199,16 +199,16 @@ const {
-| ---------------- | --------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
-| `visSize`        | Visualization size (different from canvas size on ratio-constrained visualizations, like `HeatmapVis`)                | <code>Size</code>                                     |
-| `abscissaConfig` | Abscissa configuration object passed to `VisCanvas`                                                                   | <code>AxisConfig</code>                               |
-| `ordinateConfig` | Ordinate configuration object passed to `VisCanvas`                                                                   | <code>AxisConfig</code>                               |
-| `abscissaScale`  | Computes the X coordinate in the HTML space of a given abscissa value (or the opposite with `abscissaScale.invert()`) | <code>AxisScale</code>                                |
-| `ordinateScale`  | Computes the Y coordinate in the HTML space of a given ordinate value (or the opposite with `ordinateScale.invert()`) | <code>AxisScale</code>                                |
-| `dataToWorld`    | Converts a vector from data space to world space                                                                      | <code>(vec: Vector2 &#124; Vector3) => Vector2</code> |
-| `worldToData`    | Converts a vector from world space to data space                                                                      | <code>(vec: Vector2 &#124; Vector3) => Vector2</code> |
-| `worldToHtml`    | Converts a vector from world space to HTML space                                                                      | <code>(vec: Vector2 &#124; Vector3) => Vector2</code> |
+| Name             | Description                                                                                                       | Type                                                  |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| `visSize`        | Visualization size (different from canvas size on ratio-constrained visualizations, like `HeatmapVis`)            | <code>Size</code>                                     |
+| `abscissaConfig` | Abscissa configuration object passed to `VisCanvas`                                                               | <code>AxisConfig</code>                               |
+| `ordinateConfig` | Ordinate configuration object passed to `VisCanvas`                                                               | <code>AxisConfig</code>                               |
+| `abscissaScale`  | Computes the X coordinate in data space of a given abscissa value (or the opposite with `abscissaScale.invert()`) | <code>AxisScale</code>                                |
```

Comment:
This computes the X coordinate in **world** space of a abscissa value (therefore in the **data** space).

And same below.

Reply:
:expressionless: